### ISSUE TITLE
[Question] How to change property name

### ISSUE THREAD

**Issue by siluet** [nelmio/NelmioApiDocBundle#1235]
Hi,
How can I update property name from ipAddress to ip_address?

```PHP
class User
{
    /**
     * Client Ip Address
     *
     * @var string|null
     *
     * @Assert\NotBlank()
     * @Assert\Ip(version="all")
     *
     * @SWG\Property(
     *     property="ip_address",
     *     title="ip_address",
     *     description="Client Ip Address",
     *     example="192.168.0.1",
     *     @SWG\Xml(name="ip_address", wrapped=true)
     * )
     */
    protected $ipAddress;
```

When I check documentation it shows 
```JSON
{
  "ipAddress": "192.168.0.1"
}
```
instead of ip_address.

I am using  version 3.1  without JMS Serializer.

**Comment by GuilhemN** [comment]
The support of this feature is indeed lacking. You're doing it the right way, it just isn't supported right now.

Please see https://github.com/nelmio/NelmioApiDocBundle/pull/1236.

**Comment by GuilhemN** [comment]
https://github.com/nelmio/NelmioApiDocBundle/pull/1236 is merged, so this feature will be available in 3.2.

**Comment by tung-huynh** [comment]
Hi, i am on 3.6 and cannot make this work. Any ideas?
